app/main.py

```diff
@@ -5,15 +5,18 @@
 import uuid
 import traceback
 import generate_csv_eventyay
+from svg_to_png import do_svg2png
 
 APP_ROOT = os.path.dirname(os.path.abspath(__file__))
 UPLOAD_FOLDER = os.path.join(APP_ROOT, 'static/uploads')
+SVGS_FOLDER = os.path.join(APP_ROOT, 'static/svgs')
 BADGES_FOLDER = os.path.join(APP_ROOT, 'static/badges')
 SCRIPT = os.path.join(APP_ROOT, 'merge_badges.sh')
 
 app = Flask(__name__)
 app.config['DEBUG'] = True
 app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
+app.config['SVGS_FOLDER'] = SVGS_FOLDER
```

Comment:
Please remove this line, I don't think we need this!

Reply:
This may be needed afterwards. but for now i'm removing this